src/backend/marsha/bbb/models.py

```diff
@@ -208,3 +208,47 @@ def get_source_s3_key(self, stamp=None, extension=None):
 
         stamp = stamp or to_timestamp(self.uploaded_on)
         return f"{self.classroom.pk}/classroomdocument/{self.pk}/{stamp}{extension}"
+
+
+class ClassroomRecording(BaseModel):
+    """Model representing a recording in a classroom."""
+
+    classroom = models.ForeignKey(
+        to=Classroom,
+        related_name="recordings",
+        verbose_name=_("classroom recording"),
+        help_text=_("classroom to which this recording belongs"),
+        # don't allow hard deleting a classroom if it still contains a recording
+        on_delete=models.PROTECT,
+    )
+
+    record_id = models.CharField(
+        unique=True,
+        max_length=255,
+        verbose_name=_("BBB record id"),
+        help_text=_("BBB meeting id"),
+        editable=False,
+    )
+
+    video_url = models.CharField(
```

Comment:
I choose `video_url` because it points to a mp4 file, and we would maybe store a presentation url (which points to BBB recording player) in the future.

Reply:
maybe `recording_video_url`  (not record which refers more to a piece of data in English)

Reply:
I'm don't see the value of adding `recording` for this attribute, as we are in `ClassroomRecording` class.
But I'll add it if you insist one last time @lunika @sampaccoud :)

Reply:
I failed to realize that is was a class attribute when commenting the first time. But I understand why the name video is a confusing in marsha. So I propose to call it `file_url`... because it's the file of the recording?

Reply:
WDYT about `video_file_url`, as we could also have podcast recording, and maybe other types ?

Reply:
ok for `video_file_url`

Reply:
Amen